packages/react/src/components/DataTable/__tests__/TableToolbarSearch-test.js

```diff
@@ -20,4 +20,10 @@ describe('DataTable.TableToolbarSearch', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+  it('should render without specifying an id', () => {
+    const wrapper = mount(
+      <TableToolbarSearch className="custom-class" onChange={jest.fn()} />
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
What are we looking to test in this bit? Just that no errors or warnings are logged?